container_pipeline/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+from container_pipeline.models.pipeline import Project, Build, BuildPhase
+
+
+class ProjectSerializer(serializers.HyperlinkedModelSerializer):
```

Comment:
docstrings docstrings

Reply:
That will happen i dont plan to get this in without it, as @bamachrn  stated arch discussion. This was just something i got functional over an evening, and it is where i will continue the work.